include/insns/RV32I.h

```diff
@@ -588,7 +590,7 @@ namespace SST{
           }
           R->RV32_PC += Inst.instSize;
         }else{
-          if( R->RV64[Inst.rs1] < tmp ){
+          if( (R->RV64[Inst.rs1]) < tmp64 ){
             R->RV64[Inst.rd] = 1;
           }else{
             R->RV64[Inst.rd] = 0;
```

Comment:
There was some auto-casting happening in the background that prevented this from operating as an unsigned comparison, making it explicitly uint64_t addressed that